flax/io.py

```diff
@@ -42,7 +42,8 @@ class BackendMode(Enum):
   io_mode = BackendMode.TF
 else:
   logging.warning("Tensorflow library not found, tensorflow.io.gfile "
-                  "operations will use slower native shim calls.")
+                  "operations will use slower native shim calls. "
```

Comment:
So, did you verify that native calls are actually *slower* ?

Reply:
It's more that native calls can't support GCS / other networking file systems out of the box, so IF tensorflow is installed, we'd like to just use them via the tf.io.gfile support.

Reply:
I've removed the mention of native shim calls being slower in the warning message